GMCMultiImage/GMCZoomingMultiImageView.m

```diff
@@ -212,29 +212,31 @@ - (void)updateImage {
                     }
                     
                     if (!error) {
-                        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0), ^{
+                        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
I think it's already on the main thread at this point, so we can skip this dispatch. Can you please check?